src/Desktop/UIAutomation/CustomObjects/BoolTypeConverter.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
How would you feel about moving all of the converters inside a Converters folder, and putting them into a Converters namespace? That would group them nicely together. As is, it seems a bit strange to have all the code intermixed